bindings/rust/src/lib.rs

```diff
@@ -213,6 +229,35 @@ mod tests {
         assert!(instance.is_ok());
     }
 
+    #[test]
```

Comment:
Do you think it is good idea to have tests in the same file? We have 80:20 test to implementation ratio and you will have mostly tests here.

Reply:
Well, this is the design pattern in Rust. It is possible to deviate, but this has been the usual standard.

Reply:
I get that, but in my opinion this looks good only on paper.